man/mefp.Rd

```diff
@@ -98,6 +98,17 @@ monitor.matrix(obj, X, y, verbose = TRUE)
   to \code{"fluctuation"} was used to specify it in earlier versions of
   strucchange. It still can be used now, but will be forced to \code{"RE"}
 }
+\value{
+  An object of class \code{mefp} that includes components such as:
```

Comment:
Shall we mention that this is simply a list?

Reply:
S3 objects are generally lists, I think that's not so necessary.